oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/VocabularyTreeSelectField/TreeSelectFieldModal.jsx

```diff
@@ -318,7 +326,11 @@ export const TreeSelectFieldModal = ({
                 <Button
```

Comment:
Semantically wrong element, shouldn't be a button, this is a List.Item & only button this list item should contain is the ">" to open children column. Correct DOM should look like (not exact, you can possibly use List.Content, List.Icon or List.Header inside as per SUI docs):

```
<List.Item>
  <Checkbox /> // if multiple
  {option.hierarchy.title[0]}
  <Button> // if parent node
    <Icon name="angle right" color="black" />
  </Button>
</List.Item>
```

Reply:
Using the correct elements would be much easier to do, than to convince buttons to look like list items and don't break the other buttons appearance at the same time ;)